UserTools/Configuration/Configuration.cpp

```diff
@@ -9,17 +9,21 @@ Configuration::Configuration():Tool(){}
 
 
 bool Configuration::Initialise(std::string configfile, DataModel &data){
-
   
   InitialiseTool(data);
-  InitialiseConfiguration(configfile);
-  //m_variables.Print();
+  
+  m_configfile=configfile;
+  
+  // retrieve configuration from database
+  // update Tool configuration and export to datamodel
+  // FIXME where is this run_configuration normally obtained?
+  m_data->run_configuration=6;
+  if(!LoadConfig()) return false;
```

Comment:
cant load config here as load config quiries DB for configuration and we dont know the run config number yet.

Reply:
ah, well see, that was my issue. But I think something gave me reason to believe the ToolChain couldn't just proceed without loading _some_ kind of configuration... would need to re-check what the problem was though.